### PR TITLE
Update upto 781

### PR DESCRIPTION
**Pull request by AterAnimAvis**
- [x] NSFW

### CODE REVIEW

File: quotes/data.json [hunk 1]
```diff
@@ -3617,6 +3648,207 @@
...
+            "user": "stepsword",
+            "text": "ModelPart.Cube's constructor is like code salad"
+        },
+        {
+            "user": "dpeter99",
+            "text": "You are high cause you are not underground"
+        },
+        {
+            "user": "AterAnimAvis",
+            "text": "Peter has tits?"
```
Comment (ravenbuilder934) [inline quotes/data.json:3670]: Nsfw?

File: quotes/data.json [hunk 2]
```diff
@@ -3617,6 +3648,207 @@
...
+            "text": "What's wrong with my thing?"
+        },
+        {
+            "user": "Amethyst",
+            "text": "I have the fucking hammer"
+        },
+        {
+            "user": "OakBricks",
+            "text": "no it's the <strong>condom burse</strong>"
+        },
```
Comment (ravenbuilder934) [inline quotes/data.json:3691]: nsfw?

File: quotes/data.json [hunk 3]
```diff
@@ -3617,6 +3648,207 @@
...
+            "user": "OakBricks",
+            "text": "cocaine poptart lol"
+        },
+        {
+            "user": "ravenbuilder934",
+            "text": "I'm not watching that. I, uh, don't have my earbuds in! Yeah! throws earbuds across room"
+        },
+        {
+            "user": "SizableShrimp",
+            "text": "what's wrong with ball sucking at wendy's"
```
Comment (ravenbuilder934) [inline quotes/data.json:3706]: nsfw

File: quotes/data.json [hunk 4]
```diff
@@ -3617,6 +3648,207 @@
...
+        },
+        {
+            "user": "OakBricks",
+            "side_text": "aka <strong style=\"font-size: 115%\">snek</strong>",
+            "text": "did you know: hitler breathed oxygen"
+        },
+        {
+            "user": "Simon [Lanse505]",
+            "text": "Join us Sci, we're reaching peak fluid climax"
+        },
```
Comment (ravenbuilder934) [inline quotes/data.json:3744]: nsfw

File: quotes/data.json [hunk 5]
```diff
@@ -3617,6 +3648,207 @@
...
+            "text": "make your own mapper called blackjack and hookers"
+        },
+        {
+            "user": "zml",
+            "text": "curle does know her memes"
+        },
+        {
+            "user": "FinnT730",
+            "text": "Come and fly over, and we will see if we fuck or not"
+        },
```
Comment (ravenbuilder934) [inline quotes/data.json:3768]: nsfw

File: quotes/data.json [hunk 6]
```diff
@@ -3617,6 +3648,207 @@
...
+        {
+            "user": "ProxyNeko",
+            "side_text": "about <strong style=\"font-size: 115%\">Ashers</strong>",
+            "text": "[...]you seem logical and correct[...]"
+        },
+        {
+            "user": "Amethyst",
+            "side_text": "aka <strong style=\"font-size: 115%\">Curlsed</strong>",
+            "text": "dik throbb [...] my favorite"
+        },
```
Comment (ravenbuilder934) [inline quotes/data.json:3791]: nsfw

File: quotes/data.json [hunk 7]
```diff
@@ -3617,6 +3648,207 @@
...
+        {
+            "user": "AshersLab",
+            "text": "skyjizz"
+        },
```
Comment (ravenbuilder934) [inline quotes/data.json:3795]: nsfw

File: quotes/data.json [hunk 8]
```diff
@@ -3617,6 +3648,207 @@
...
+        {
+            "user": "Amethyst",
+            "text": "i was wondering how much money that dude gets paid per dick"
```
Comment (ravenbuilder934) [inline quotes/data.json:3798]: nsfw

File: quotes/data.json [hunk 9]
```diff
@@ -3617,6 +3648,207 @@
...
+        },
+        {
+            "user": "ProxyNeko",
+            "text": "Thought this said \"sugar cookie tits\", got excited there"
```
Comment (ravenbuilder934) [inline quotes/data.json:3802]: nsfw?

File: quotes/data.json [hunk 10]
```diff
@@ -3617,6 +3648,207 @@
...
+            "user": "Amethyst",
+            "text": "I've acquired an aaafaaaaa"
+        },
+        {
+            "user": "CJDaBomb",
+            "text": "Who wouldn't want to be Canadian"
+        },
+        {
+            "user": "LagoCraft",
+            "side_text": "aka <strong style=\"font-size: 115%\">Bohdan</strong>",
```
Comment (ravenbuilder934) [inline quotes/data.json:3814]: nsfw? maybe?

File: quotes/data.json [hunk 11]
```diff
@@ -3617,6 +3648,207 @@
...
+            "text": "Do you think you will go faster in water slide if you use lube as Water ? <img class=\"emote\" src=\"https://discord.com/assets/263a7f4eeb6f69e46d969fa479188592.svg\" alt=\":thinking:\"/>"
+        },
+        {
+            "user": "CJDaBomb",
+            "text": "Higher than everyone"
+        },
+        {
+            "user": "PlatinPython",
```
Comment (ravenbuilder934) [inline quotes/data.json:3822]: nsfw

File: quotes/data.json [hunk 12]
```diff
@@ -3617,6 +3648,207 @@
...
+        {
+            "user": "AshersLab",
+            "text": "who the hell wears pants in bed"
+        },
+        {
+            "user": "Amethyst",
+            "text": "it's my slapping zone"
+        },
+        {
+            "user": "ravenbuilder934",
```
Comment (ravenbuilder934) [inline quotes/data.json:3834]: nsfw?

**Review by sciwhiz12 — APPROVED** [review]
:shipit: :cookie:

**Pull request merged**